src/Umbraco.Cms.Api.Management/ViewModels/Document/DocumentConfigurationResponseModel.cs

```diff
@@ -10,5 +10,6 @@ public class DocumentConfigurationResponseModel
 
     public required bool AllowNonExistingSegmentsCreation { get; set; }
 
+    [Obsolete("Use DocumentTypeConfigurationResponseModel.ReservedFieldNames from the DocumentTypeConfiguration endpoint instead.")]
```

Comment:
Good catch I've updates the messages to reference the controllers, however I haven't added the "Scheduled for removal in V16." since these are used in the REST API, so they won't necessarily be removed in V16, but instead when we decide to make a V2 of the specific API.

Reply:
Makes sense 👍